README.md

```diff
@@ -9,41 +7,25 @@
 3. [Setup - The basics of getting started with haproxy](#setup)
     * [Beginning with haproxy](#beginning-with-haproxy)
 4. [Usage - Configuration options and additional functionality](#usage)
-    * [Configuring haproxy options](#configuring-haproxy-options)
-    * [Configuring an HAProxy daemon listener](#configuring-haproxy-daemon-listener)
-    * [Configuring HAProxy load-balanced member nodes](#configuring-haproxy-loadbalanced-member-nodes)
-    * [Configuring a load balancer with exported resources](#configuring-a-load-balancer-with-exported-resources)
-    * [Classes and Defined Types](#classes-and-defined-types)
-        * [Class: haproxy](#class-haproxy)
-        * [Defined Type: haproxy::balancermember](#defined-type-haproxybalancermember)
-        * [Defined Type: haproxy::backend](#defined-type-haproxybackend)
-        * [Defined type: haproxy::frontend](#defined-type-haproxyfrontend)
-        * [Defined type: haproxy::listen](#defined-type-haproxylisten)
-        * [Defined Type: haproxy::userlist](#define-type-haproxyuserlist)
-        * [Defined Type: haproxy::peers](#define-type-haproxypeers)
-        * [Defined Type: haproxy::peer](#define-type-haproxypeer)
```

Comment:
Good question. I talked this over with @jbondpdx and we're adding back the ToC links for the Usage subsections, but leaving out the sections that are now part of the Reference section. (Reference has its own mini-ToC.)